build/yamls/flow-visibility/patches/chmonitor/chMonitor.yml

```diff
@@ -0,0 +1,28 @@
+- op: add
```

Comment:
Is this change independent from the rest of this PR (PersistentVolume support)? If yes, could it go into a separate PR?

Reply:
It is inspired by the PV support. This change allows us to generate a manifest without the ClickHouse monitor. We suppose this may happen when user have a large enough PV storage. In that case, the monitor won't be triggered as the throughput bottleneck would be in the flow aggregator side instead of the ClickHouse storage space side. But overall it make sense to me to put it in a separate PR.

Reply:
This part has been removed from this PR now. Hope it is more clear now. Thanks!